### PR TITLE
Support absolute paths to templates

### PR DESCRIPTION
**Pull request by abrown-sg**
Previously, if a user tried to specify a path to a template file, it would fail because the template loader was only able to look within the package.  This adds support for looking on the filesystem then in the package.

### CODE REVIEW

**Comment by retr0h** [comment]
:+1: 

This looks like a good solution!

File: molecule/utilities.py [hunk 1]
```diff
@@ -76,8 +80,19 @@ def write_template(src, dest, kwargs={}, _module='molecule', _dir='templates'):
...
-    template = env.get_template(src)
+    path = os.path.dirname(src)
+    filename = os.path.basename(src)
+
+    # template file doesn't exist
+    if path and not os.path.isfile(src):
+        print('\n{}Unable to locate template file: {}{}'.format(Fore.RED, src, Fore.RESET))
+        sys.exit(1)
+
+    # look for template in filesystem, them molecule package
```
Comment (rgreinho) [inline molecule/utilities.py]: them -> then

**Comment by rgreinho** [comment]
:+1:  I did not know about the Choiceloader, that's pretty cool!

**Comment by retr0h** [comment]
> I did not know about the Choiceloader, that's pretty cool!

Yeah, me either.  Pretty sweet.

**Pull request merged**